dipy/denoise/noise_estimate.py

```diff
@@ -117,7 +120,8 @@ def piesno(data, N, alpha=0.01, l=100, itermax=100, eps=1e-5, return_mask=False)
                                                           itermax=itermax,
                                                           eps=eps,
                                                           return_mask=True,
-                                                          initial_estimation=initial_estimation)
```

Comment:
If you are going to have parameters spread over multiple lines, I prefer to have one line per parameter:

    _piesno_3D(data[..., idx, :],
                        N,
                        alpha=alpha,
                        l=l,
                        itermax=itermax,
                        eps=eps,
                        return_mask=True,
                        initial_estimation=initial_estimation)